test_files/es6/optional.js

```diff
@@ -0,0 +1,2 @@
+function OptionalArgument(/** number */ x, /** string= */ y) {
```

Comment:
I guess it's up to the original author to do this though, right, so it's not sickle's problem?  Or do you just mean our test cases ought to match Closure style?